### ISSUE TITLE
argument --optimizer: invalid choice: 'adam_ood' (choose from 'adam_vanilla', 'adam_masker')

### ISSUE THREAD

**Issue by MrSworder** [alinlab/MASKER#7]
In readme, there is a command :
python train.py --dataset review --split_ratio 0.25 --seed 0 \
    --train_type base \
    --backbone bert --classifier_type softmax --optimizer adam_ood \

But in the code, train.py, there is no option adam_ood.
